src/neph/parser/parser.cr

```diff
@@ -44,16 +44,8 @@ module Neph
 
       job = Job.new(job_name, job_command, parent_job)
       job.dir = job_config["dir"].as(String) if job_config.has_key?("dir")
-      job.ignore_error = if job_config["ignore_error"].as(String) == "true"
-                           true
-                         else
-                           false
-                         end if job_config.has_key?("ignore_error")
-      job.hide = if job_config["hide"].as(String) == "true"
-                   true
-                 else
-                   false
-                 end if job_config.has_key?("hide")
+      job.ignore_error = job_config["ignore_error"] ? true : false if job_config.has_key?("ignore_error")
+      job.hide = job_config["hide"] ? true : false if job_config.has_key?("hide")
```

Comment:
What happen when I specify ignore_erro/hide as "false"?
I think
- `job_config.has_key?("ignore_error")` => `true`
- `job_config["ignore_error"]` => `true` (Since job_config["ignore_error"] is not nil)

Am I misunderstanding? 🤔

Reply:
[If as suffix](https://crystal-lang.org/docs/syntax_and_semantics/as_a_suffix.html); [ternary if](https://crystal-lang.org/docs/syntax_and_semantics/ternary_if.html).
When the condition of the `if` suffix is true; it evaluates the other code on the left side.
So, when `job_config` has the `ignore_error` key, it evaluates and assigns its value to `job.ignore_error`.
I have tried it (modified `hide_command` to `false` in `sample/neph.yml`), and it works as expected.